src/serializer/ResidualHeapVisitor.js

```diff
@@ -113,20 +113,25 @@ export class ResidualHeapVisitor {
     }
   }
 
-  visitObjectProperties(obj: ObjectValue): void {
+  visitObjectProperties(obj: ObjectValue, kind?: ObjectKind): void {
     // visit properties
-    for (let [symbol, propertyBinding] of obj.symbols) {
-      invariant(propertyBinding);
-      let desc = propertyBinding.descriptor;
-      if (desc === undefined) continue; //deleted
-      this.visitDescriptor(desc);
-      this.visitValue(symbol);
+    if (kind !== "ReactElement") {
+      for (let [symbol, propertyBinding] of obj.symbols) {
+        invariant(propertyBinding);
+        let desc = propertyBinding.descriptor;
+        if (desc === undefined) continue; //deleted
+        this.visitDescriptor(desc);
+        this.visitValue(symbol);
+      }
     }
 
     // visit properties
-    for (let propertyBinding of obj.properties.values()) {
-      invariant(propertyBinding);
-      this.visitObjectProperty(propertyBinding);
+    for (let [propertyBindingKey, propertyBindingValue] of obj.properties) {
+      if (kind === "ReactElement" && (propertyBindingKey === "$$typeof" || propertyBindingKey === "_owner")) {
+        continue;
```

Comment:
Why is this special case needed? The `$$typeof` symbol is an intrinsic and so it should be fine visiting.

The `_owner` probably does need to be serialized if it is set to anything other than `null` (but ideally it should always be `null` here).

I'm not sure why you need any special cases here?

Reply:
Same as above